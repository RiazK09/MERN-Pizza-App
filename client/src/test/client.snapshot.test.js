import React from "react";
// Imported the Loading component for testing.
import Loading from "../components/Loading";
// Imported react-test-renderer for rendering snapshots.
import renderer from "react-test-renderer";

/* Snapshot Test
• After renderer.create, I passed in the component that I want to render.
• I then called .toJSON() on it to get a JSON representation of the component. */

it("renders correctly", () => {
  const tree = renderer.create(<Loading />).toJSON();
  expect(tree).toMatchSnapshot();
});
