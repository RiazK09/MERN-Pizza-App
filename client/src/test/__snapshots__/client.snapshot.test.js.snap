// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="text-center"
>
  <div
    className="spinner-border"
    role="status"
    style={
      Object {
        "height": "100px",
        "marginTop": "100px",
        "width": "100px",
      }
    }
  >
    <span
      className="sr-only"
    >
      Loading...
    </span>
  </div>
</div>
`;
